WordPress/WordPressTest/MediaPicker/MediaLibraryPickerDataSourceTests.swift

```diff
@@ -73,6 +74,39 @@ class MediaLibraryPickerDataSourceTests: CoreDataTestCase {
         self.waitForExpectations(timeout: 5, handler: nil)
     }
 
+    func testBroadcastMediaUpdates() throws {
+        contextManager.useAsSharedInstance(untilTestFinished: self)
+
+        MediaBuilder(context)
+            .with(remoteStatus: .sync)
+            .with(blog: blog)
+            .with(title: "avatar")
+            .with(mediaType: .image)
+            .build()
+        contextManager.saveContextAndWait(context)
+
+        dataSource.setMediaTypeFilter(.image)
+        var changes = 0
+        dataSource.registerGroupChangeObserverBlock {
+            changes += 1
+        }
+        expect(self.dataSource.numberOfAssets()).to(equal(1))
+
+        contextManager.performAndSave { context in
+            let media = try? context.fetch(Media.fetchRequest()).first as? Media
+            XCTAssertNotNil(media)
+            media?.title = "avatar"
+        }
+        expect(changes).toEventually(equal(0))
```

Comment:
I should've mentioned this in the PR description, if we think it's okay to let FRC receive updates even if the properties values aren't really updated (like this test case here), then this PR isn't necessary and can be closed. cc @jkmassel